src/functions.js

```diff
@@ -0,0 +1,27 @@
+export const stringSumCalculator = (input) => {
+  if (input === "") return 0;
+
+  //basic delimiter
+  let delimiter = /,|:/;
+
+  //custom delimiter
+  if (input[0] === "/" && input[0] === "/") {
```

Comment:
해당 라인의 조건이 조금 이상한 것 같은데.. 오타인가요..? `//;\n1;2;3`을 파악하기 위함이였다면,
```suggestion
  if (input[0] === "/" && input[1] === "/") {
```
이렇게 쓰는게 맞을 것 같아요 🤔

Reply:
헉 오타가 났었네요,,, 알려주셔서 감사합니다!!